src/features/todo/index.tsx

```
@@ -0,0 +1,191 @@
+          </Flex>
+        </>
+      )}
+    </div>
+  );
+
+  const renderDetail = () => {
+    if (mode === 'create') {
+      return (
+        <div className="TodoDetailContainer flex flex-1 height-100">
+          <TodoCreate />;
+        </div>
```

Biome (1.9.4)

[error] There is a suspicious semicolon in the JSX element.

This is usually the result of a typo or some refactor gone wrong.
Remove the semicolon, or move it inside a JSX element.

(lint/suspicious/noSuspiciousSemicolonInJsx)